src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1292,8 +1292,7 @@ public AsyncJobStatusDTO refreshPools(String ownerKey, Boolean autoCreateOwner,
 
         JobConfig config = RefreshPoolsJob.createJobConfig()
             .setOwner(owner)
-            .setLazyRegeneration(true)
-            .setForceUpdate(forceUpdate != null && forceUpdate);
+            .setLazyRegeneration(true);
```

Comment:
Same as the other comment, just want to make sure we are intentionally keeping this new line

Reply:
The lazy regeneration wasn't added as a result of this logic, it was simply a refactor to move it out of the refresher's constructor and into more of a builder pattern. That pattern has been retained and extended slightly with Ondrej's work to break some of those circular dependencies.